datashuttle/tui/configs.py

```diff
@@ -198,6 +207,9 @@ def on_mount(self) -> None:
             id = "#configs_name_input"
             self.query_one(id).tooltip = get_tooltip(id)
 
+            # Register event handler for project name input
+            self.query_one(id).changed_callback = self.on_input_changed
```

Comment:
Due to textual magic, `self.on_input_changed` will be automatically be called when an input is changed (no need to set it explicitly)